build/mvn

```diff
@@ -38,7 +38,7 @@ install_app() {
   # setup `curl` and `wget` silent options if we're running on Jenkins
   local curl_opts="-L"
   local wget_opts=""
-  if [ -n "$AMPLAB_JENKINS" ]; then
+  if [ -n "$AMPLAB_JENKINS" -o -n "$K8S_JENKINS" ]; then
```

Comment:
That has a bunch of other side effects that I'm not entirely sure we want -- definitely open to trimming this down before merging.

I'm still experimenting to try to get http://spark-k8s-jenkins.pepperdata.org:8080/job/PR-spark-k8s-unit-tests-SBT-TESTING working